packages/passwordless-auth-construct/src/custom-auth/custom_auth_service.ts

```diff
@@ -12,7 +12,12 @@ import {
   SignInMethod,
 } from '../types.js';
 import { CognitoMetadataKeys } from '../constants.js';
+import {
+  AdminUpdateUserAttributesCommand,
+  CognitoIdentityProviderClient,
+} from '@aws-sdk/client-cognito-identity-provider';
 
+const PASSWORDLESS_SIGNUP_ATTR_NAME = 'custom:_passwordless_signup';
```

Comment:
*nit:* Sign up is either two words as a verb or hyphenated (sign-up) as a noun/adjective. Either way it should be split up.

```suggestion
const PASSWORDLESS_SIGNUP_ATTR_NAME = 'custom:_passwordless_sign_up';
```